build-support/bin/get_added_files.sh

```diff
@@ -6,4 +6,4 @@
 # Check for copies (-C) and moves (-M), so we don't get false positives when people do
 # refactorings. -l50 bounds the time git takes to search for these non-additions.
 # See git-diff(1) and https://stackoverflow.com/a/2299672/2518889 for discussion of these options.
-exec git diff --cached --name-only --diff-filter=A -C -M -l50
+exec git --no-pager diff --cached --name-only --diff-filter=A -C -M -l50
```

Comment:
Depending on how the user has their git set up, this script would sometimes fail. For example, my git for some reason defaults to opening `git diff` through `less`.

Now, the script will work no matter what.

Reply:
I didn't know about this argument! Note that many programs (including `git`) will respect the `PAGER` env var, and I've usually done this by running the command with `PAGER=cat` -- but this is definitely better!